SU-SWS/decanter
# 195 added all Stanford approved fonts

# READY FOR REVIEW

# Summary
- Added all Stanford approved fonts (cleaned up files, etc. of existing fonts, too)

# Needed By (Date)
- End of week

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Compile styleguide (`grunt styleguide`)
3. Verify that all fonts and style variations are rendering

# Affected Projects or Products
- Decanter

# Associated Issues and/or People
- #195 
- #220 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
